Makefile

```diff
@@ -0,0 +1,2 @@
+cli:
+	goreleaser release --snapshot --rm-dist
```

Comment:
Maybe add a note about this in the README?

(Does this mean my little `build-cli.sh` bash script isn't needed now?)